index.html

```diff
@@ -17,11 +18,17 @@ <h1 class="welcome-title"> Welcome to Real Dev Squad </h1>
     <h2 class="info-note highlighted-text">You can check out all our current events happening <a href="./events.html">here</a></h2>
     <img class="logo-img" src="img/Real-Dev-Squad@1x.png" alt="Real Dev Squad logo" />
     <h3 class="info-note"> To know how this page is getting developed, join our Discord Channel. Contact one of the existing members </h3>
+    <div class="cta-button__container " id = "buttonDiv">
+      <h4 align="center">Please subscribe for further event updates</h4>
```

Comment:
AFAIK: 
HTML align attribute supports col, colgroup, tbody, td, tfoot, th, thead, tr elements.
Usage of align attribute for any other HTML elements is deprecated; You must use CSS for those

Please confirm